File: Java/car.java

class car {

    Integer id ;
    String license;
    Account driver;
    private Integer passenger;

    public car(String license,Account driver){
        this.license = license;
        this.driver = driver;
        
    }
    
    void printDatacar(){
        if (passenger != null ){
            System.out.println("license: "+ license + " Name Driver: " + driver.name + " Passanger: " + passenger);
        }
        
    }
    public Integer getPassenger(){
        return passenger;
    }
    public void setPassenger(Integer passenger){
        if (passenger==4){
            this.passenger= passenger;
        }else{
            System.out.println("Necesitas asignar 4 pasajeros");
        }

    }

   

    

}

File: PHP/car.php
<?php

class car{

    public $id;
    public $license;
    public $driver;
    public $passengers;

    public function __construct($license, $driver){
        $this->license = $license;
        $this->driver= $driver;
    }

    public funtion printDataCar() {
        echo "licencia; $this->license Driver :".$this->driver->name
    }
    
}
?>